dhall/tests/format/duplicatesB.dhall

```diff
@@ -0,0 +1,6 @@
+{ foo.bar = 1
+, foo.baz
+  = {- Comments within values are still preserved-}
+    2
```

Comment:
This is unrelated to this PR, but makes me realize that maybe the `=` should rather be on the same line as the dotted fields:

```suggestion
, foo.baz =
    {- Comments within values are still preserved-}
    2
```

Having the `=` below the keys is IMHO mostly warranted when the keys are spread over several lines and especially when the last key has a trailing comment, e.g.

```
{ k
  {- bla -}
  = v
}
```

Thoughts? (We can move this discussion to a separate issue though.)

Reply:
@sjakobi: It should probably be a separate issue.  I try to keep changes logically separate if possible

Reply:
I opened https://github.com/dhall-lang/dhall-haskell/issues/2051 to track this.